src/lava/magma/runtime/runtime.py

```diff
@@ -220,7 +226,7 @@ def _run(self, run_condition):
                                     actors.join()
                                     if actors.exception:
                                         _, traceback = actors.exception
-                                        print(traceback)
+                                        self.log.debug(traceback)
```

Comment:
Is this really debug?
I think errors should always be thrown unconditionally (or in this case the remote error messages).